src/index.ts

```diff
@@ -13,6 +13,6 @@ app.listen(
   port,
   host,
   () => {
-    logger.info(`Server running on ${host}:${port}`);
+    logger.info(`Server running on http://${host}:${port}`);
```

Comment:
This seems unrelated but I am OK with letting this small change ride along here.